datafusion/core/tests/sql/group_by.rs

```diff
@@ -194,7 +194,7 @@ async fn csv_query_group_by_and_having_and_where() -> Result<()> {
 async fn csv_query_having_without_group_by() -> Result<()> {
     let ctx = SessionContext::new();
     register_aggregate_csv(&ctx).await?;
-    let sql = "SELECT c1, c2, c3 FROM aggregate_test_100 HAVING c2 >= 4 AND c3 > 90";
+    let sql = "SELECT c1, c2, c3 FROM aggregate_test_100 where c2 >= 4 AND c3 > 90";
```

Comment:
I think we can remove the whole test as we definitely have coverage for selecting with predicates in the where clause and the name of the test doesn't make sense `csv_query_having_without_group_by`